app/2.0/start/first-element/step-2.md

```diff
@@ -91,7 +92,7 @@ element. If the element has a `<dom-module>`, this script is usually placed
 <em>inside</em> the `<dom-module>` to keep everything together.
 
 
-Starting code—element registration { .caption }
+Starting `code—element` registration { .caption }
```

Comment:
That's an em dash, not a hyphen. The is the starting code, and this part deals with element registration.

We could change all of these em dashes to colon if you think it'd be clearer:

Starting code: HTML imports
Starting code: local DOM template  
Starting code: element registration

I think on the page the em dashes are pretty clear, but they look weird in markdown because the code font em dash is more or less identical to a hyphen.